### PR TITLE
Add support for pretty print json output.

### PR DESCRIPTION
**Pull request by rsjones**

### CODE REVIEW

File: lib/response-handler.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 module.exports = function responseHandler (req, res, statusCode, payload) {
+  console.log('sending response', req.query)
```
Comment (haoliangyu) [inline lib/response-handler.js:2]: Please remove the console.log.

**Review by haoliangyu — COMMENTED** [review]
Is it possible to add a test for this change?

**Review by haoliangyu — APPROVED** [review]
LGTM

**Comment by haoliangyu** [comment]
@rsjones Let us know if you need a release any time soon.

**Comment by rgwozdz** [comment]
I'll plan to release by end of week.

**Pull request merged**